pyroclast/common/datasets.py

```diff
@@ -0,0 +1,38 @@
+mapping = {}
+
+
+def register(name):
+
+    def _thunk(func):
+        mapping[name] = func
+        return func
+
+    return _thunk
+
+
+# Implement datasets here
```

Comment:
Example?

Reply:
I have literally nothing to use as an example. Sorry.